odoo/addons/base/models/ir_actions.py

```diff
@@ -271,7 +271,11 @@ def _check_view_mode(self):
                             help="Model name of the object to open in the view window")
     target = fields.Selection([('current', 'Current Window'), ('new', 'New Window'), ('inline', 'Inline Edit'), ('fullscreen', 'Full Screen'), ('main', 'Main action of Current Window')], default="current", string='Target Window')
     view_mode = fields.Char(required=True, default='tree,form',
-                            help="Comma-separated list of allowed view modes, such as 'form', 'tree', 'calendar', etc. (Default: tree,form)")
+                            help="Comma-separated list of allowed view modes, such as 'form', 'tree', 'calendar', etc. "
+                                 "(default: tree,form). The first view mode is used by default.")
+    default_mobile_mode = fields.Char(string="Default Mobile View", help="View mode override in mobile and small "
+                                             "screen environments. If unset, will default to kanban view "
```

Comment:
> If unset, will default to kanban view

Why is it set explicitly everywhere then? :thinking:

Reply:
This change was made before that the spec is changed in the ticket with this part: 
`use a runbot to find out which window actions should have a default_mobile_mode set to kanban, and set it in xml files
basically: all window actions that have kanban as a possible view mode (either through the view_mode field or through the view_ids​ field but where kanban is not the first mode`